plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StandardFacetInstallDelegate.java

```diff
@@ -148,6 +148,9 @@ void createConfigFiles(IProject project, IProgressMonitor monitor)
     if (appEngineWebXml.exists()) {
       return;
     }
+    if (!webInfDir.exists()) {
+      ResourceUtils.createFolders(webInfDir, progress.newChild(1));
+    }
```

Comment:
Something seems off. We call `ResourceUtils.createFolders(webInfDir, ...)` above.